toolbox/fdc3-workbench/src/components/Header.tsx

```diff
@@ -74,6 +77,7 @@ export const Header = (props: { fdc3Available: boolean }) => {
 					<div>
 						<Typography variant="h3" color="inherit" className={classes.fdc3}>
 							<img src="./fdc3-logo.svg" className={classes.headerCube} />
+							&nbsp;{`{workbench}`}
```

Comment:
This bit is the only part I have an issue with. I'll ping gwendolyn and see if we can just get the name added to the logo.

We can improve in a later PR